go.mod

```diff
@@ -1,7 +1,7 @@
 module github.com/terraform-providers/terraform-provider-azurerm
 
 require (
-	github.com/Azure/azure-sdk-for-go v31.0.0+incompatible
+	github.com/Azure/azure-sdk-for-go v31.1.0+incompatible
```

Comment:
If we need to update the SDK that should be done in a separate PR, i can open that and get it merged for you once 1.33 goes out 🙂

Reply:
Thanks, that sounds great!